docs/adr/0035-refactor-payments-frontend.md

```diff
@@ -0,0 +1,68 @@
+# Refactor Subscription Platform frontend
+
+- Status: proposed
+- Deciders: Ben Bangert
+- Date: 2023-03-07
+
+## Context and Problem Statement
+
+The Subscription Platform frontend has become overly complex, making it inflexible and difficult to work with. This has resulted in longer development times on new features and in some cases left us unable to easily accommodate requests from relying parties.
+
+Additionally, as the app has grown, so have the number of network calls causing a reduction in app performance.
+
+Finally, including some application logic on the frontend, has made it more complex than necessary. As a result it is harder to maintain, and less flexible and harder to change.
+
+To address the problems briefly discussed above, and to meet current and future feature and relying party requests refactoring of the existing Subscription Platform is necessary.
+
+## Decision Drivers
+
+- Faster development - This applies to both the refactoring necessary for SubPlat 3.0, as well as future relying party and feature requests
+- Easier routing - The SubPlat teams wants to perform more advanced routing to allow features such as, Relying Party specific checkout pages, A/B testing, etc,
+- GraphQL support - Improve app performance and responsivness by reducing the number of network requests
+- Reuse existing components - We have received generally favourable reviews of our existing frontend look and feel, and want to keep the appearance the same
+
+## Considered Options
```

Comment:
We did a comparison with Gatsby, would be good to include that and why we didn't use it.